README.md

```diff
@@ -1,28 +1,147 @@
-# Kubernetes cluster-api-provider-vsphere Project
+# Kubernetes Cluster API Provider vSphere
 
-This repository hosts a concrete implementation of a provider for vsphere for the [cluster-api project](https://github.com/kubernetes-sigs/cluster-api).
+[![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/cluster-api-provider-vsphere)](https://goreportcard.com/report/github.com/kubernetes-sigs/cluster-api-provider-vsphere)
 
-## Community, discussion, contribution, and support
+Kubernetes-native declarative infrastructure for vSphere.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+## What is the Cluster API Provider vSphere
 
-* Join our Cluster API vSphere Provider sessions
-  * Bi-weekly on Mondays @ 1:00 PT on [Zoom](https://zoom.us/j/875399243) starting on 8/13
-  * Previous meetings: \[ [notes](https://docs.google.com/document/d/1jQrQiOW75uWraPk4b_LWtCTHwT7EZwrWWwMdxeWOEvk/edit?usp=sharing) \]
+The [Cluster API][cluster_api] brings
+declarative, Kubernetes-style APIs to cluster creation, configuration and
+management. Cluster API Provider vSphere is a concrete implementation of
+Cluster API for vSphere.
 
-You can reach the maintainers of this project at:
+The API itself is shared across multiple cloud providers allowing for true vSphere
+hybrid deployments of Kubernetes. It is built atop the lessons learned from
+previous cluster managers such as [kops][kops] and
+[kubicorn][kubicorn].
 
-* [Slack](http://slack.k8s.io/): #cluster-api
-* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+## Launching a Kubernetes cluster on vSphere
 
-### Code of conduct
+Check out the [getting started guide](docs/getting-started.md) for launching a
+cluster on vSphere.
 
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+## Features
 
-### Getting Started
+- Native Kubernetes manifests and API
+- Manages the bootstrapping of VMs on cluster.
+- Choice of Linux distribution between Ubuntu 18.04 and CentOS 7
+  using VM Templates based on [OVA images](docs/machine_images.md).
+- Deploys Kubernetes control planes into provided clusters on vSphere.
+- Doesn't use SSH for bootstrapping nodes.
+- Installs only the minimal components to bootstrap a control plane and workers.
 
-See the [Getting Started](docs/getting_started.md) guide to get up and going with Cluster API for vSphere.
+---
 
-### Troubleshooting
+## Compatibility with Cluster API and Kubernetes Versions
 
-Please see the [Troubleshooting](docs/troubleshooting.md) guide if having problems using Cluster API for vSphere.
+This provider's versions are compatible with the following versions of Cluster API:
+
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (unreleased)|
+|-|-|-|
+|vSphere Provider v1alpha1 (v0.3)|✓|☓|
+|vSphere Provider v1alpha1 (v0.4)|✓|☓|
```

Comment:
Hi @ykakarap,

Please modify the table to be:

||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (unreleased)|
|-|-|-|
| CAPV v1alpha1 (v0.3)|✓|☓|
| CAPV v1alpha1 (v0.4)|✓|☓|
| CAPV v1alpha1 (master)|☓|✓|